content/swift/concepts/data-types/data-types.md

```diff
@@ -49,7 +49,7 @@ let verySmallInteger: Int8 = 16
 
 ## Floating Point Numbers
 
-A `Float` and a `Double` are number data types that allow for decimals. A `Float` is a 32-bit 'floating-point' number and a `Double` is a 64-bit floating-point number, that being said, a `Float` has approaximately half as much precision as a `Double`. If you need a high precision of accuracy, it is best to use a `Double`. In fact, when you declare a variable without specifying the type, Swift will type inference a `Double` as a precaution.
+A `Float` and a `Double` are number data types that allow for decimals. A `Float` is a 32-bit 'floating-point' number and a `Double` is a 64-bit floating-point number, that being said, a `Float` has approximately half as much precision as a `Double`. If you need a high precision of accuracy, it is best to use a `Double`. When a variable is declared without a specified type, Swift will type inference a `Double` as a precaution.
 
```

Comment:
@yangc95 After looking into this, I'm suggesting we remove the comment in lines # 63-66. The live entry as it is now is rendering the \```swift snippet with the multi-line string just fine.

cc: @lucwhite